frontend/src/contexts/global/global-actions.ts

```diff
@@ -20,30 +20,32 @@ export function initAppAction() {
     try {
       const isInit = await Service.IsAppInitialised()
 
-      if (IS_TEST_MODE && !isInit) {
+      if (!isInit) {
```

Comment:
Hm I don't think we should call IniitaliseApp here. The init _should_ happen in the onboard flow.. 🤔

Reply:
I think the previous condition was correct. Only initialize if not already initialized AND in test mode. That way for tests we can navigate to different pages regardless of whether you are initialized without triggering the onboarding flow.

Otherwise if you aren't initialized always go to onboarding..

Reply:
Hm I'm not sure on my above comment looking into I more I can see why the change is necessary.